python/samples/03.adaptiveCards.a.typeAheadBot/teamsapp.local.yml

```diff
@@ -63,19 +63,6 @@ provision:
       appPackagePath: ./appPackage/build/appPackage.${{TEAMSFX_ENV}}.zip
 
 deploy:
-  # prerequisite: pip install
-  - uses: script
```

Comment:
this should be exist so that devs don't need to run the install themselves to run the sample.

Reply:
Python extension does not use the newly created virtual environment. Manual step is required to run the app. So running the commands here makes little sense.
I adds an extra step in readme asking users to run poetry install by themselves and select the new venv when Python extension displays a notification.

Does this makes sense to you? The original way will always fail the local debug if user does not switch the venv in time - the debug process does not stop and ask users to select a venv.

Reply:
Yesterday I pushed a fix for this process, now if you clean your environment and run TTK it should always run properly.

Reply:
As synced offline, please help update this PR after you tuned the config to enable 1-click F5 experience. This sample enables 1-click F5 experience for a sample in Microsoft-Teams-Samples code: https://github.com/frankqianms/bot-conversation-py/tree/main. Could be helpful to your work.